src/app/splashscreen/splashscreen.component.scss

```diff
@@ -0,0 +1,5 @@
+@import "../styles/components/headings";
+
+.logo {
+    max-width: 25%;
```

Comment:
We are expecting Images and color from Peace Corps, This image is just a snip of the prototype document. I think doing it now will be useless. I will keep this in mind when we get the final images.